### PR TITLE
Fix 0.8 fixture for `prf` capability

### PR DESCRIPTION
**Pull request by expede**

### CODE REVIEW

**Comment by expede** [comment]
Fixes https://github.com/ucan-wg/spec/issues/68

File: fixtures/0.8.1/valid.json [hunk 1]
```diff
@@ -204,7 +204,7 @@
         "exp": 4804143412,
         "att": [
           {
-            "with": "prf/0",
+            "with": "prf:0",
```
Comment (Gozala) [inline fixtures/0.8.1/valid.json:207]: Would not this affect the actual token ? Or was this just a typo here ?

Comment (Gozala) [reply fixtures/0.8.1/valid.json:207]: As far as I can tell `token` field is also needs to be updated a per https://github.com/ipld/js-dag-ucan/runs/6458958822?check_suite_focus=true#step:5:107

Comment (Gozala) [reply fixtures/0.8.1/valid.json:207]: Pasting token into https://ucan.xyz/validator also confirmed my suspicion
image

Comment (matheus23) [reply fixtures/0.8.1/valid.json:207]: Yeah, you're right. I don't know how to recreate the signature correctly with that issuer's key, so here's a different token that should do the same thing:
```
eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJhdWQiOiJkaWQ6a2V5Ono2TWtnWDVqalJVYnR5c2dnRTRyYUNhcUNYODhBelN2WXE4MVdKa0JvQTFvdDhhZSIsImF0dCI6W3sid2l0aCI6InByZjowIiwiY2FuIjoidWNhbi9ERUxFR0FURSJ9XSwiZXhwIjo0ODA0MTQzNDEyLCJmY3QiOltdLCJpc3MiOiJkaWQ6a2V5Ono2TWtoejNhdENZdGk5QktWUEc4alcyYlFCbnR5RW5ZS3BpTVJreTJuOGg0eVY1VSIsInByZiI6W119.TTqaclWZgPWDIjGk4tdHPHhzUE2NOLjGh7Wk3SX4GCQy9Kw3niD-9Q4ubTv2BoCk_anNEmCrmWUCijBd137pBg
```

I'll edit the PR.

Comment (matheus23) [reply fixtures/0.8.1/valid.json:207]: Actually, scrap that. The token I pasted above is incorrect, and I now see where the tokens are generated: https://github.com/ucan-wg/ucan-fixture-gen/blob/main/src/samples.ts

**Comment by matheus23** [comment]
Unfortunately, it's impossible to regain access to the original private key behind the fixtures, so the diff is larger than neccessary. I've used the ucan-fixture-gen project to re-generate these fixtures correctly (and I've opened a PR over there https://github.com/ucan-wg/ucan-fixture-gen/pull/1).

There was another fixture using an incorrect delimiter, I've fixed that one, too. :v:

Should be good now.

**Pull request merged**